apps/meteor/app/api/server/v1/chat.ts

```diff
@@ -364,19 +364,13 @@ API.v1.addRoute(
 				throw new Meteor.Error('error-messageid-param-not-provided', 'The required "messageId" param is missing.');
 			}
 
-			const msg = await Messages.findOneById(this.bodyParams.messageId);
-
-			if (!msg) {
-				throw new Meteor.Error('error-message-not-found', 'The provided "messageId" does not match any existing message.');
-			}
-
```

Comment:
I think this is a breaking change, we are changing the error messages. Since we are validating it only inside the function, the error is different.

Reply:
We should also add a e2e test for this (if it doesn't exist)

Reply:
There should be tests for chat.react endp.

I'm a bit in conflict on whether or not this is a breaking change 🤔 tho. Error won't be the exact same but will happen on the exact same condition, and the status code will be the same too. So, didn't saw this as a breaking change.

Can leave it back but that also introduces another DB call (that we don't actually need)